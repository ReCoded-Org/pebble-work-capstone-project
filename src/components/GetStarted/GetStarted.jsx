import React from "react";

import Button from "../Button/Button";
const GetStarted = () => {
    return (
        <div className='m-2  flex  flex-col justify-center gap-2 py-48'>
            <div className='mx-4 text-center'>
                <h1 className=' flex items-center justify-center text-4xl font-bold md:px-52 lg:px-52'>
                    Become the change you wish to see in the world.
                </h1>
                <p className='mt-5 md:px-52 lg:px-52'>
                    Body placeholder for text paragraph. A paragraph is a
                    self-contained unit of text dealing with a particular point
                    or idea.
                </p>
            </div>
            <div className='flex flex-row items-center justify-center'>
                <Button
                    label='Get started'
                    textColor='text-white'
                    bgColor='bg-secondary-300'
                    borderColor='border-secondary-300'
                    height='h-10'
                />
                <button className='m-2 flex rounded p-2 hover:bg-primary-200 hover:text-white hover:shadow-xl'>
                    Who are we?
                    <svg
                        xmlns='http://www.w3.org/2000/svg'
                        className='h-6 w-6'
                        fill='none'
                        viewBox='0 0 24 24'
                        stroke='currentColor'
                        strokeWidth='2'
                    >
                        <path
                            strokeLinecap='round'
                            strokeLinejoin='round'
                            d='M9 5l7 7-7 7'
                        />
                    </svg>
                </button>
            </div>
        </div>
    );
};

export default GetStarted;
