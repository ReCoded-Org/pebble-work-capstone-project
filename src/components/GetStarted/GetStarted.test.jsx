import React from "react";
import renderer from "react-test-renderer";

import GetStarted from "./GetStarted";

it("renders correctly", () => {
    const tree = renderer.create(<GetStarted />).toJSON();
    expect(tree).toMatchSnapshot();
});
