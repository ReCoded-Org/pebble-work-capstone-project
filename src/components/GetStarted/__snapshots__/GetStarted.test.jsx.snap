// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="m-2 flex flex-col justify-center gap-2 py-48"
>
  <div
    className="mx-4 text-center"
  >
    <h1
      className=" flex items-center justify-center text-4xl font-bold md:px-52 lg:px-52"
    >
      Become the change you wish to see in the world.
    </h1>
    <p
      className="mt-5 md:px-52 lg:px-52"
    >
      Body placeholder for text paragraph. A paragraph is a self-contained unit of text dealing with a particular point or idea.
    </p>
  </div>
  <div
    className="flex flex-row items-center justify-center"
  >
    <button
      className="mt-1 mb-1 w-30 h-10 undefined bg-secondary-300 undefined undefined hover:undefined text-white font-normal  py-1  px-2 border-secondary-300 hover:undefined rounded  hover:shadow-lg undefined"
      onClick={[Function]}
      type="button"
    >
      Get started
    </button>
    <button
      className="m-2 flex rounded p-2 hover:bg-primary-200 hover:text-white hover:shadow-xl"
    >
      Who are we?
      <svg
        className="h-6 w-6"
        fill="none"
        stroke="currentColor"
        strokeWidth="2"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9 5l7 7-7 7"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
</div>
`;
